android/src/main/java/com/opentokreactnative/utils/Utils.java

```diff
@@ -100,6 +101,25 @@ public static List<IceServer> sanitizeIceServer(ReadableArray serverList) {
         return iceServers;
     }
 
+    public static ArrayList<VideoTransformer> sanitizeVideoTransformerList(PublisherKit publisher, ReadableArray transformerList) {
+        ArrayList<VideoTransformer> nativeVideoTransformers = new ArrayList<>();
+        if (transformerList != null) {
```

Comment:
Nit: if you check for this first, you can bail early (e.g., not have to instantiated `nativeVideoTransformers`).